cypress/e2e/models/migration/reports/reports.ts

```diff
@@ -72,10 +72,11 @@ export function expandArticle(name: string): void {
 
 export function verifyApplicationRisk(risktype: string, appName: string): void {
     // Verifies particular application's risk type
-    selectItemsPerPageAdoptionCandidate(100);
+    selectArticleItemsPerPage(100, adoptionCandidateDistributionTitle);
```

Comment:
Do you really need to keep separate function for choosing amount of items per page in select article? Consider to reuse basic `selectItemsPerPage()` function here

Reply:
because in the reports page there are multiple articles and at each article we can choose items per "page" i had to create separate method. for example:
![image](https://github.com/konveyor/tackle-ui-tests/assets/132557033/9c5bebdd-8034-41d0-898b-8ea9784186d2)